core/docker/default/etc/jvm.config

```diff
@@ -12,3 +12,6 @@
 -XX:PerBytecodeRecompilationCutoff=10000
```

Comment:
In commit message mention that it improves S3 performance a lot on graviton instances

> It was not enabled by default in the backport because of
conservative approach.

Mention that:
* It's enabled in Java 18 by default, which is already released. Therefore risk is minimal